### PR TITLE
Update readme to use Magewell and not use blackmagic

### PR DESCRIPTION
**Pull request by PotatoPeeler3000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,62 +10,54 @@ IHMC Robot Data Logger
...
-        - You do not need the SDK, just the plain Desktop Video product. The registration has a "Download only" link in the bottom left to bypass.
-    - Untar Desktop video: `tar xzvf Blackmagic_Desktop_Video_Linux_12.1.tar.gz`
-    - Install debian packages: `sudo dpkg -i Blackmagic_Desktop_Video_Linux_12.1/deb/x86_64/desktopvideo_12.1a9_amd64.deb`
-    - Possible run `sudo apt --fix-broken install` to install missing dependencies.
-- (If logging video streams with capture card) Update Blackmagic firmware for each Decklink card (first card is 0, second 1, etc).
-    - `BlackmagicFirmwareUpdater update [Decklink card]`
+- Capture cards to record video feeds (optional)
+  - [Magewell Pro Capture Card](https://www.magewell.com/products/pro-capture-sdi)
+  - BlackMagic Decklink Mini Recorder Capture Card (Deprecated)
+    - This is deprecated because it only works on Ubuntu 20.04 and is no longer supported by the maintainers of this repository
```
Comment (ds58) [inline README.md]: Can you clarify that "it" refers to https://github.com/ihmcrobotics/ihmc-java-decklink-capture and not the actual hardware. Correct me if I am wrong

File: README.md [hunk 2]
```diff
@@ -11,62 +10,54 @@ IHMC Robot Data Logger
...
+
+## Ubuntu 22.04 (recommended)
+- Install Ubuntu Desktop 22.04 LTS (its convenient to have a GUI to work with)
```
Comment (ds58) [inline README.md]: "its" -> "it's"

File: README.md [hunk 3]
```diff
@@ -11,62 +10,54 @@ IHMC Robot Data Logger
...
+---
 
 ## Setting up cameras
 
-The easiest way is to use the configuration application from `gradle deploy`. If you want to manually do the setup, follow these steps.
+This file holds the information regarding the cameras that the logger will try to capture the feeds from.
+It is possible to configure the cameras from the `gradle deploy` but since they get changes often its important to know how to change it manually.
 
-Create a new file ~/.ihmc/CameraSettings.yaml. A basic setup looks like this
+Create a new file ~/.ihmc/CameraSettings.yaml. A basic setup looks like this:
```
Comment (ds58) [inline README.md]: Put ~/.ihmc/CameraSettings.yaml in backticks

**Review by ds58 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PotatoPeeler3000) [reply README.md]: Ooooo good call!

File: README.md (hunk 2 above)
Comment (PotatoPeeler3000) [reply README.md]: Yep

File: README.md (hunk 3 above)
Comment (PotatoPeeler3000) [reply README.md]: Yes sir!

**Review by ds58 — APPROVED** [review]

**Pull request merged**